tgui/jest.config.js

```diff
@@ -5,7 +5,7 @@ module.exports = {
   testEnvironment: 'jsdom',
   testRunner: require.resolve('jest-circus/runner'),
   transform: {
-    '^.+\\.(js|jsx|cjs|ts|tsx)$': require.resolve('babel-jest'),
```

Comment:
Removing jsx?

Reply:
didnt really need to be there, I can add it back though

Reply:
done